### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by Hvskyy**

### CODE REVIEW

**Review by Hopefuls — APPROVED** [review]
LGTM lol

File: index.html [hunk 1]
```diff
@@ -75,6 +75,7 @@ <h5>Yikes</h5>
 <h5>Pog</h5>
 <h5>Weary</h5>
 <h5>Stinky-poo</h5>
+ <h5>nani</h5>
```
Comment (Hopefuls) [inline index.html:78]: Actually, have it uppercase, looks better (`Nani`)

**Review by Hopefuls — CHANGES_REQUESTED** [review]

**Review by Xyoyu — CHANGES_REQUESTED** [review]
Make it propercase pls

**Comment by Hvskyy** [comment]
changed it

**Review by Xyoyu — APPROVED** [review]

**Pull request closed without merging**